types/prettier/index.d.ts

```diff
@@ -365,7 +365,7 @@ export interface IntSupportOption extends BaseSupportOption<'int'> {
 }
 
 export interface IntArraySupportOption extends BaseSupportOption<'int'> {
-    default: number[];
+    default: Array<{ value: number[] }>;
```

Comment:
I'm not 100%, but I think this should be:

```ts
{ value: number }[]
```

Right? The value shouldn't itself be an array I don't think.

Reply:
Did you see my context?
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/48670#issue-500975322